packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2093,8 +2097,12 @@ export class ContainerRuntime
 
 		// Do shallow copy of message, as the processing flow will modify it.
 		const messageCopy = { ...messageArg };
+		const pretendLocal = this.stashClientIds.has(messageArg.clientId);
```

Comment:
could we do something where we set the local client id to the oldest stashed ops client id, and then change local client id each time the original client's id changed? Basically, always pretend to be the same client as the old client?

You might even be able to do it with just the set you've already created, and just watch the quorum clients for changes, and when even it changes to one in the set, change the member here. You will always need to set the initial client id.

I'm hoping that would help prevent the need to duplicate the saved ops, but not sure yet.

Reply:
i think i may have missed something. is this change applying all ops as stashed ops first, and then this is basically acking them by combining save with pending, and only the pending may not be acked, but we re-apply them all anyways to the local client processes them all as local ops?

Reply:
Yes. I'm stashing sequenced ops along with pending ones and applying both at their reference sequence numbers.